### PR TITLE
Task-6 priceRangeSorting completed

### PR DESCRIPTION
**Pull request by nirav-improwised**
Completed Task-6 : Display list of restaurant based on given Price Range with Sorting - passing range as argument like 100-500 and sorting with argument like low-high

Run: node PriceRangeSorting.js {100-500} {low-high}

Output: 
image

### CODE REVIEW

File: priceRangeSorting.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const filePriceSorting = require("./modules/priceSorting.js")
+
+
+let arr = process.argv[2].split("-");
+const lowerLimit = parseFloat(arr[0]);
+const upperLimit = parseFloat(arr[1]);
+
+const sortingOrder = process.argv[3];
+
+console.log(lowerLimit)
```
Comment (monikagorkhiya) [inline priceRangeSorting.js:11]: ?

File: priceRangeSorting.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+console.log(upperLimit)
```
Comment (monikagorkhiya) [inline priceRangeSorting.js:12]: ?

File: priceRangeSorting.js (hunk 1 above)
Comment (nirav-improwised) [reply priceRangeSorting.js]: Wrote it for my understanding while writing the code, Removed it.

File: priceRangeSorting.js (hunk 2 above)
Comment (nirav-improwised) [reply priceRangeSorting.js]: Removed console.log(upperLimit) and console.log(lowerLimit).

**Pull request merged**